policyengine_us/parameters/gov/states/co/tax/income/credits/eitc/match.yaml

```diff
@@ -27,3 +17,61 @@ metadata:
       href: https://tax.colorado.gov/sites/tax/files/documents/DR_0104_Book_2022.pdf#page=16 
     - title: 2023 Colorado Individual Tax Filing Guide 104 Book, Line 5
       href: https://tax.colorado.gov/sites/tax/files/documents/Book0104_2023.pdf#page=18
+    - title: Colorado Revised Statutes, 39-22-119. Expenses related to child and dependent care refundable credit against state tax - tax preference performance statement - definition - repeal. (1.7).(a)&(b)
+      href: https://leg.colorado.gov/sites/default/files/2024a_1134_signed.pdf#page=3
+
+brackets:
+  - threshold:
+      2021-01-01: 0
+    amount:
+      2021-01-01: 0.1
```

Comment:
I dont think we should have to duplicate the amounts but rather make varying thresholds since it is flat until 2025 if I am reading this correctly